core/jazz_cognito-token-authorizer/.jshintrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
we dont usually check in jshintrc files. I would prefer we add jshintrc and edtior.config files as part of a story where we have discussed our coding standards. Till then lets delete this file and add the appropriate entry in .gitignore.

Reply:
Removed .jshintrc file